SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -406,23 +409,33 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	const FFilterPredicate ActorFilter = [](const Worker_EntityId, const SpatialGDK::EntityViewElement& Element) {
 		return !Element.Components.ContainsByPredicate(SpatialGDK::ComponentIdEquality{ SpatialConstants::TOMBSTONE_COMPONENT_ID });
 	};
+	const FFilterPredicate TombstoneFilter = [](const Worker_EntityId, const SpatialGDK::EntityViewElement& Element) {
```

Comment:
Where is this used ?

Reply:
should have been used in the tombstone subview- will update (although won't break anything, just means too many entities are being passed through that subview)

Reply:
actually no! The reason it was removed from the subview is because there's two different types of tombstoning- so we added a new tag instead, and made that the query tag for the subview. Deleting the filter instead

Reply:
removed now